app/controllers/api/base_controller/normalizer.rb

```diff
@@ -112,7 +112,8 @@ def normalize_select_attributes(obj, opts)
       end
 
       def normalize_array(obj)
-        obj.collect { |item| normalize_attr(@req.subcollection || @req.collection, item) }
+        type = @req.subcollection || @req.collection
```

Comment:
If you wish, you can use `type = @req.subject` here ☺️

Reply:
👍 🎵